confetti.js

```diff
@@ -9,7 +9,12 @@
 // t.ex. baserat på slump eller baserat på inparametern groupName
 // så kan det istället bli en blandning av färger.
 const chooseParticleColor = (groupName) => {
-  return 'gray'
+  let letters = '0123456789ABCDEF';
+  let color = '#';
+  for (var i = 0; i < 6; i++) {
+    color += letters[Math.floor(Math.random() * 16)];
+  }
+  return color;
```

Comment:
En fin lösning det här, man kan få alla tillgängliga färger, snyggt!

Alternativer till den här typ av lösning är att man kan välja returnera RGB värden och jobba direkt med siffror, som i RGB(0, 0, 0); 

Till exempel
```
return `RGB(${Math.floor(Math.random() * 255)}, ${Math.floor(Math.random() * 255)}, ${Math.floor(Math.random() * 255)})`
```

Eller för hex värden så har Nummer i javascript en toString metod där man kan skicka in 16 för att konvertera till hex. Så Number(255).toString(16) ger 'f'. Då kan man även returnera en del färgkod baserad på ett slumpat värde mellan 0 och 16777215 som är max antal olika färger man kan få från RGB 0, 0, 0 till RGB 255, 255, 255.

Det kan da se såhär ut
`return '#' + Math.floor(Math.random()*16777215).toString(16)`